apps/frontend/src/components/ChessBoard.tsx

```diff
@@ -39,363 +40,393 @@ export function isPromoting(chess: Chess, from: Square, to: Square) {
   }
 
   return chess
-    .history({ verbose: true })
+    .moves({ verbose: true })
     .map((it) => it.to)
     .includes(to);
 }
 
-export const ChessBoard = memo(({
-  gameId,
-  started,
-  myColor,
-  chess,
-  board,
-  socket,
-  setBoard,
-}: {
-  myColor: Color;
-  gameId: string;
-  started: boolean;
-  chess: Chess;
-  setBoard: React.Dispatch<
-    React.SetStateAction<
-      ({
-        square: Square;
-        type: PieceSymbol;
-        color: Color;
-      } | null)[][]
-    >
-  >;
-  board: ({
-    square: Square;
-    type: PieceSymbol;
-    color: Color;
-  } | null)[][];
-  socket: WebSocket;
-}) => {
-  console.log("chessboard reloaded")
-  const { height, width } = useWindowSize();
+export const ChessBoard = memo(
+  ({
+    gameId,
+    started,
+    myColor,
+    chess,
+    board,
+    socket,
+    setBoard,
+  }: {
+    myColor: Color;
+    gameId: string;
+    started: boolean;
+    chess: Chess;
+    setBoard: React.Dispatch<
+      React.SetStateAction<
+        ({
+          square: Square;
+          type: PieceSymbol;
+          color: Color;
+        } | null)[][]
+      >
+    >;
+    board: ({
+      square: Square;
+      type: PieceSymbol;
+      color: Color;
+    } | null)[][];
+    socket: WebSocket;
+  }) => {
+    console.log('chessboard reloaded');
```

Comment:
can we remove the console log?

Reply:
i tried to fix the lint issues using yarn format but the main branch doesn't have all files formatted so it changed all the formatting what should i do?